tasks/image/build.go

```diff
@@ -77,7 +77,7 @@ func buildIsStale(ctx *context.ExecuteContext, t *Task) (bool, error) {
 	excludes = append(excludes, ".dobi")
 
 	mtime, err := fs.LastModified(&fs.LastModifiedSearch{
-		Root:     t.config.Context,
```

Comment:
This doesn't seem like it can be right. The last modified time should be based on the context directory, not the working directory.